tests/kernel/usb_nic.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: usb_nic
+# Summary: Simple smoke test for testing USB NIC connected to system
+# Maintainer: LSG QE Kernel <kernel-qa@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    my ($self) = @_;
+
+    select_serial_terminal;
+
+    my $interface = script_output 'basename $(readlink /sys/class/net/* | grep usb )';
+
+    assert_script_run "echo \"BOOTPROTO='dhcp'\" > /etc/sysconfig/network/ifcfg-$interface";
+    assert_script_run "echo \"STARTMODE='auto'\" >> /etc/sysconfig/network/ifcfg-$interface";
+
+    systemctl("restart wicked.service", 300);
+
+    my $ping_peer = script_output "ip route show dev $interface | cut -d ' ' -f 7";
```

Comment:
Are you sure you want to ping the USB NIC itself instead of e.g. the default gateway (or any other remote host)?

Reply:
It is good enough check that usb system is working, but we can extend it  if really needed.

Reply:
I don't think the ping will actually touch the USB subsystem at all. You might as well ping `localhost` explicitly.

Reply:
If eth2 has ip, it means, that it all is working. It gets ip from dhcp over the network.

Reply:
Then we can simplify the check to `ip addr show dev $interface` and skip the pointless ping.

Reply:
OK

Reply:
I will change it to ping default gw.

Reply:
Let's wait for new VR: https://openqa.suse.de/tests/14016025

Reply:
New set of VR s after changing ping peer to gateway: https://openqa.suse.de/tests/14023221#next_previous